optimade/filtertransformers/mongo.py

```diff
@@ -58,7 +58,26 @@ def not_implemented_string(self, value):
 
     def value_list(self, arg):
         # value_list: [ OPERATOR ] value ( "," [ OPERATOR ] value )*
-        raise NotImplementedError
+        # NOTE: no support for optional OPERATOR
+        values = []
+        for value in arg:
+            if value.startswith('"') and value.endswith('"'):
+                value = value[1:-1]
+            if str(value) in self.operator_map.keys():
+                raise NotImplementedError(
```

Comment:
I guess you could return a normal `[ OPERATOR ] value` here, checking against the length of `arg`, since I assume it _is_ implemented for a single operator and value, but not multiple - or?

Reply:
This is meant to block cases like `filter=dimension_types HAS ANY > 1`, which we don't have implemented. What filter would your suggestion above support?

Reply:
Ah, I thought this was implemented. I only thought the actual continuous list of operators weren't implemented, e.g. `filter=dimension_types HAS ANY > 1, < 1` (which is of course the same as just `= 1`, but in any case it shows what I thought :)

Reply:
Ah I see, I'll check this and see what we can get away with